tests/resources/helper_functions.py

```diff
@@ -446,3 +447,113 @@ def create_nft_erc20_with_pool(
     pool_token = ERC20Token(web3, pool_address)
 
     return bpool, erc20_token, erc721_token, pool_token
+
+
+def join_pool_with_max_base_token(bpool, web3, base_token, wallet, amt: int = 0):
+    """
+    Join pool with max base token if amt is 0. Otherwise join pool with amt base token.
+    """
+    pool_token_out_balance = bpool.get_balance(
+        base_token.address
+    )  # pool base token balance
+    max_out_ratio = bpool.get_max_out_ratio()  # max ratio
+
+    max_out_ratio_limit = int(from_wei(max_out_ratio, "ether") * pool_token_out_balance)
```

Comment:
Instead of using `eth_utils.from_wei` from , can you instead use `ocean_lib.web3_internal.currency.from_wei`
https://github.com/oceanprotocol/ocean.py/blob/v4main/ocean_lib/web3_internal/currency.py#L40-L46

This eliminates the need for the "ether" argument and leads to more consistency throughout the ocean.py tests.